src/base/anim/interpolated.h

```diff
@@ -33,9 +34,13 @@ template <typename Type> class Weighted
 		return value == other.value && weight == other.weight;
 	}
 
+	template <typename Comparator =
```

Comment:
`Comparator` could be named `Less`

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/555/commits/fafd45e1db663964cf1ac0b117b926414f0524d7